### PR TITLE
Propagate correct replica count to rancher chart

### PR DESCRIPTION
**Pull request by saahildhulla**
Need to check/test -- meant to draft this but it wouldn't let me correct it

### CODE REVIEW

File: main.tf [hunk 1]
```diff
@@ -23,6 +23,7 @@ resource "null_resource" "ansible_playbook" {
         --user=${var.ssh_username} \
         --ssh-common-args='-o StrictHostKeyChecking=no ${local.ansible_ssh_proxy}' \
         -e 'cert_names=${local.cert_names}' \
+        -e ip_addr_count=length('${local.ip_addresses}')
```
Comment (steved) [inline main.tf]: I think this is `${length(var.node_ips)}`?

File: ansible/roles/rancher/tasks/rancher.yml [hunk 2]
```diff
@@ -4,5 +4,5 @@
   when: "'rancher-stable' not in repos.stdout"
 
 - name: Install Rancher
-  command: "helm install rancher-stable/rancher --name rancher --namespace cattle-system --version {{ rancher_version }} --set tls=external --set privateCA=true --set addLocal=false --set auditLog.level=1 --description='RanchHand Deploy' --wait"
-  when: "'rancher' not in charts.stdout"
\ No newline at end of file
+  command: "helm install rancher-stable/rancher -f values.yml --name rancher --namespace cattle-system --version {{ rancher_version }} --set tls=external --set privateCA=true --set addLocal=false --set auditLog.level=1 --description='RanchHand Deploy' --wait"
```
Comment (steved) [inline ansible/roles/rancher/tasks/rancher.yml]: I think this values.yml needs to be placed on the system with a new sub-task

File: ansible/roles/rancher/tasks/rancher.yml [hunk 3]
```diff
@@ -4,5 +4,16 @@
   when: "'rancher-stable' not in repos.stdout"
 
 - name: Install Rancher
-  command: "helm install rancher-stable/rancher --name rancher --namespace cattle-system --version {{ rancher_version }} --set tls=external --set privateCA=true --set addLocal=false --set auditLog.level=1 --description='RanchHand Deploy' --wait"
-  when: "'rancher' not in charts.stdout"
\ No newline at end of file
+  command: "helm install rancher-stable/rancher -f values.yml
```
Comment (zs-ddl) [inline ansible/roles/rancher/tasks/rancher.yml]: You'll need to use proper YAML support for multi-line strings

```suggestion
  command: <-
    helm install rancher-stable/rancher -f values.yml
```

File: ansible/roles/rancher/tasks/values.yml [hunk 4]
```diff
@@ -0,0 +1,3 @@
+antiAffinity: required
+
+replicas: ${ip_addr_count}
```
Comment (steved) [inline ansible/roles/rancher/tasks/values.yml]: ✂️ this file now?

Comment (saahildhulla) [reply ansible/roles/rancher/tasks/values.yml]: Yep!

**Pull request merged**